resources/settings.xml

```diff
@@ -59,20 +59,22 @@
-        <setting label="30923" help="30924" type="action" action="RunScript(script.module.inputstreamhelper, info)" enable="eq(-2,true)"/> <!-- InputStream Helper information -->
-        <setting label="30925" help="30926" type="action" option="close" action="Addon.OpenSettings(script.module.inputstreamhelper)" enable="eq(-3,true)" subsetting="true"/> <!-- InputStream Helper settings -->
-        <setting label="30927" type="lsep"/> <!-- Logging -->
-        <setting label="30929" help="30930" type="enum" id="max_log_level" lvalues="30430|30431|30432|30433" default="0"/>
+        <setting label="30901" type="lsep"/> <!-- InputStream Adaptive -->
+        <setting label="30903" type="action" visible="!System.HasAddon(inputstream.adaptive)" enable="false"/>
+        <setting label="30905" help="30906" type="bool" id="useinputstreamadaptive" default="true" visible="System.HasAddon(inputstream.adaptive)"/>
+        <setting label="30907" help="30908" type="action" option="close" action="Addon.OpenSettings(inputstream.adaptive)" visible="System.HasAddon(inputstream.adaptive) + [String.StartsWith(System.BuildVersion,18) | String.StartsWith(System.BuildVersion,19)]" enable="eq(-1,true)" subsetting="true"/> <!-- InputStream Adaptive settings -->
+        <setting label="30909" help="30910" type="action" action="RunScript(script.module.inputstreamhelper, info)" enable="eq(-2,true)"/> <!-- InputStream Helper information -->
+        <setting label="30911" help="30912" type="action" option="close" action="Addon.OpenSettings(script.module.inputstreamhelper)" enable="eq(-3,true)" subsetting="true"/> <!-- InputStream Helper settings -->
+        <setting label="30913" type="lsep"/> <!-- Cache -->
+        <setting label="30915" help="30916" type="action" action="RunPlugin(plugin://plugin.video.vrt.nu/tokens/delete)"/> <!-- Delete tokens -->
+        <setting label="30917" help="30918" type="action" action="RunPlugin(plugin://plugin.video.vrt.nu/favorites/refresh)"/> <!-- Refresh favorites -->
+        <setting label="30919" help="30920" type="action" action="RunPlugin(plugin://plugin.video.vrt.nu/resumepoints/refresh)"/> <!-- Refresh resumepoints -->
+        <setting label="30921" help="30922" type="bool" id="usemenucaching" default="true"/>
+        <setting label="30923" help="30924" type="bool" id="usehttpcaching" default="true"/>
+        <setting label="30925" help="30926" type="action" action="RunPlugin(plugin://plugin.video.vrt.nu/cache/delete)" enable="eq(-1,true)" subsetting="true"/>
+        <setting label="30927" help="30928" type="slider" id="httpcachettldirect" default="5" range="1,1,480" option="int" enable="eq(-2,true)" subsetting="true"/>
+        <setting label="30929" help="30930" type="slider" id="httpcachettlindirect" default="60" range="1,1,480" option="int" enable="eq(-3,true)" subsetting="true"/>
```

Comment:
Does it make sense that a user can set both types of cache to 480 minutes?

Reply:
Probably not, but the only reason to do this is to make both sliders visually comparable. Not sure if that is a must. I haven't fully thought about the use-cases really.

Also, 480 minutes (6 hours) is a questionable max value. Probably 4 hours or maybe 2 hours is sufficient. (Unless people do want very long values, in theory they could specify both ttl values that expires only at midnight) In fact, we probably should expire caches at midnight.

And a per minute step is also questionable for the indirect value.

Reply:
I turned it down to 240 minutes, if there is ever a use-case to increase it, we can still do it.

I also was wondering why we still needed the option to turn HTTP caching off, as you can set both to 0, but this is easier *and* we do have some caching included that are not tied to 'direct' and 'indirect'.